src/main/java/racingcar/controller/WebController.java

```diff
@@ -0,0 +1,37 @@
+package racingcar.controller;
+
+import java.util.List;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import racingcar.dto.RequestDto;
+import racingcar.dto.ResponseDto;
+import racingcar.model.Cars;
+import racingcar.service.GameService;
+import racingcar.util.NameFormatConverter;
+
+@Controller
+public class WebController {
+
+    private final GameService gameService;
+
+    @Autowired
+    public WebController(GameService gameService) {
+        this.gameService = gameService;
+    }
+
+    @PostMapping("/plays")
+    public ResponseEntity<ResponseDto> playGame(@RequestBody RequestDto requestDto) {
+        List<String> carNames = NameFormatConverter.splitNameByDelimiter(requestDto.getNames());
+        Cars cars = new Cars(carNames);
+
+        gameService.executeRacingGame(cars, requestDto.getCount());
+
+        final ResponseDto responseDto = new ResponseDto(cars.getWinners(), cars.getCars());
+
+        return ResponseEntity.ok()
+                .body(responseDto);
```

Comment:
`@ResponseBody` vs `ResponseEntity`

1번 질문에 답이 되어버릴 것 같지만, `@RestController`를 사용하게되면, `@ResponseBody`의 어노테이션 사용도 불필요해집니다.

```suggestion
      return responseDto;
```

즉, 지금의 코드 구조를 유지한 채 위와 같이 변경할 수 있습니다. 코드의 가독성 측면에서  `@ResponseBody` 를 사용합니다. 물론 이 또한 팀마다 컨벤션이 다를 수 있습니다 :)

Reply:
완태가 말씀하신대로 헤더와 상태 코드 설정이 불필요한 상황이라면, `@RestController`를 사용하고 응답에 적절한 Dto 객체를 바로 반환하는 게 가독성 측면에서 더 좋을 것 같네요! 감사합니다!!

Reply:
넵! 이 두가지 방식외에도 상태 코드, 헤더를 설정할 수 있는 다양한 방식이 있으니 함께 학습해보시면 좋을 것 같습니다.

https://www.baeldung.com/spring-response-header

Reply:
내용 확인했습니다.
`HttpServletResponse.addHeader`와 `Filter` 인터페이스를 구현함으로써 헤더를 설정할 수 있네요!
Reactive Endpoint에서의 헤더 설정에 대해서도 언급하는듯 하네요..! 이에 대해서는 아직 무지하기 때문에 다음에 학습해보도록 하겠습니다 ㅎㅎ!
링크 첨부해주셔서 감사합니다 :)